lint-compose/src/main/kotlin/team/duckie/quackquack/lint/compose/NamedArgumentDetector.kt

```diff
@@ -106,6 +106,6 @@ class NamedArgumentDetector : Detector(), SourceCodeScanner {
         }
     }
 
-    private fun ASTNode.isValueArgumentName() =
+    private fun ASTNode.isNotValueArgumentName() =
```

Comment:
```suggestion
    private fun ASTNode.isNotValueArgumentName() = elementType != KtNodeTypes.VALUE_ARGUMENT_NAME
```

으로 바뀌어야 하는거 아닌가요? 👀 
지금처럼이라면 메서드 이름은 `VALUE_ARGUMENT_NAME` 가 아닌지 나타내는데 실제 구현은 `VALUE_ARGUMENT_NAME` 가 맞는지에요